fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java

```diff
@@ -4209,6 +4209,101 @@ public void testConcurrentPuts() throws InterruptedException, IOException {
         }
     }
 
+    @Test
+    public void testConcurrentPutsAsChildren() throws InterruptedException, IOException {
+        final String parent = getRandomUniqueId();
+        executeAndClose(putObjMethod(parent));
+        final String first = parent + "/1";
+        final String second = parent + "/2";
+        final String third = parent + "/3";
+        final String fourth = parent + "/4";
+        final Thread t1 = new Thread(() -> {
+            executeAndClose(putObjMethod(first));
+        });
+        final Thread t2 = new Thread(() -> {
+            executeAndClose(putObjMethod(second));
+        });
+        final Thread t3 = new Thread(() -> {
+            executeAndClose(putObjMethod(third));
+        });
+        final Thread t4 = new Thread(() -> {
+            executeAndClose(putObjMethod(fourth));
+        });
+        t1.start();
```

Comment:
If you want these threads to fire as concurrently as possible, then you should use a `Phaser` (like some of the other tests), or a `CountDownLatch`.

Reply:
Yeah I saw that after the fact. I'll fix it.